### ISSUE TITLE
Fine tune sequence length

### ISSUE THREAD

**Issue by Alexmhack** [kingoflolz/mesh-transformer-jax#112]
Hi @kingoflolz, I am confused on what is the sequence length of my dataset. [Learning Rate Notes](https://github.com/kingoflolz/mesh-transformer-jax/blob/master/howto_finetune.md#learning-rate-notes) on `howto_finetune.md` mentions the below

> Your .tfrecord files should have a number in the file name indicating how many sequences are in the dataset. Divide that number by the batch size and the result is how many steps are in an epoch. Now we can write the schedule.

I created tfrecord files using [create_tfrecords.py on gpt-neo](https://github.com/EleutherAI/gpt-neo/blob/master/data/create_tfrecords.py) and my tfrecord files name looks like `github_0_0_100000.tfrecords`. My doubt is what is the sequence length in this filename??

One more question(sorry if it doesn't belong to this issue) how to configure the fine tune schedule so that model is trained on each token only once.

**Comment by kingoflolz** [comment]
in that script the chunk_size argument is the sequence length (default 2048).

Follow the last paragraph in the learning rate notes to configure one epoch training.
